src/core/css/inc/mixins.scss

```diff
@@ -67,3 +69,119 @@
       + 'Please make sure it is defined in `$breakpoints` map.';
   }
 }
+
+/* Arrows */
+@mixin arrow($margin: 3px, $type: 'previous') {
+  background: url('~amo/img/icons/arrow.svg') 50% 50% no-repeat;
+  border-radius: $border-radius-m;
+  content: '';
+  display: inline-block;
+  height: 16px;
+  margin-right: -$margin * 2;
+  position: relative;
+  right: -$margin;
+  top: 4px;
+  width: 22px;
+
+  @if $type == 'previous' {
+    left: -$margin;
+    margin-left: -$margin * 2;
+    margin-right: 0;
+    right: auto;
+    transform: rotate(180deg);
+  }
+
+  [dir=rtl] & {
+    left: -$margin;
+    margin-left: -$margin * 2;
+    margin-right: 0;
+    right: auto;
+    transform: rotate(180deg);
+
+    @if $type == 'previous' {
+      left: auto;
+      margin-left: 0;
+      margin-right: -$margin * 2;
+      right: -$margin;
+      transform: none;
+    }
+  }
+}
+
+/* Button */
+@mixin button($background: $button-background-color,
```

Comment:
Would a `Button` component work here too? That might be nicer than including the mixin in CSS each time but I'm not sure.